.env

```diff
@@ -0,0 +1,7 @@
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgres
+POSTGRES_DB=um_db
+# MIGRATE_DIRECTION can be "up" for direct migration or "down" to reverse migration
```

Comment:
WDYT about moving comments to README.md?

Reply:
Fix it